oak_channel/src/basic_framed.rs

```diff
@@ -14,12 +14,30 @@
 // limitations under the License.
 //
 
+//! Basic framed format for use before the channel is handed of to the
+//! application.
+//!
+//! This protocol is used once.
+//! 1. To load the application. Here the host is the sender, enclave is the
+//!    recipient.
+//! 2. To return the attestation evidence. Here the host is the recipient,
+//!    enclave is the sender.
+//!
+//! The protocol is very simple:
+//! 1. sender sends the size of the payload, as u32
+//! 2. sender sends MAX_SIZE of data and waits for the recipient to acknowledge
+//! 3. recipient reads up to MAX_SIZE of data and acks by responding with the
+//!    amount of data read
+//! 4. repeat (2) and (3) until all the data has been transmitted
```

Comment:
This solution is not pretty, but the alternative of using full micro_rpc for this exchange isn't either. 

The reason is that then we'd need use channels in the orchestrator, to escape from the micro_rpc server implementation back to starting the application after it has loaded. 

We can't start it from within an RPC handler, since the orchestrator is single threaded. It'd never send a response.